src/App.js

```diff
@@ -1,79 +1,16 @@
 import React from 'react';
 import './App.css';
 
+import { Welcome } from './components/welcome/Welcome';
+import { Header } from './components/header/Header';
+import { Article } from './components/article/Article';
```

Comment:
```suggestion
import { Welcome } from './components/Welcome/Welcome';
import { Header } from './components/Header/Header';
import { Article } from './components/Article/Article';
```
https://mateacademy.slack.com/archives/G011BRK1WBH/p1592988955011600